packages/blade/src/components/TopNav/TopNav.web.tsx

```diff
@@ -73,21 +40,19 @@ const TopNav = ({ children, ...styledProps }: TopNavProps): React.ReactElement =
     <TopNavContext.Provider value={{ backgroundColor: styledProps.backgroundColor }}>
       <Box
         display="grid"
-        gridTemplateColumns={{ base: 'minmax(0, 1fr) auto', m: 'auto minmax(0, 1fr) auto' }}
+        gridTemplateColumns="auto minmax(0, 1fr) auto"
         alignItems="center"
         position="sticky"
```

Comment:
why do we use position sticky over position fixed?

Reply:
with position fixed the topnav will pop out of the layout tree and won't reserve space causing it to overlap with the dashboard layout. 

<img width="778" alt="image" src="https://github.com/user-attachments/assets/fd1c120f-4bba-4d53-b940-166b6d2b8395">


While sticky will both make it so it sticks like position:fixed + reserve layout space.